libwallet/assets_config.go

```diff
@@ -156,6 +156,16 @@ func (mgr *AssetsManager) SetCurrencyConversionExchange(xc string) {
 	}()
 }
 
+// ExchangeRateFetchingEnabled returns true if privacy mode isn't turned on and
+// a valid exchange rate source is configured.
+func (mgr *AssetsManager) ExchangeRateFetchingEnabled() bool {
+	if mgr.IsPrivacyModeOn() {
+		return false
+	}
+	xc := mgr.GetCurrencyConversionExchange()
+	return xc != "" && xc != values.DefaultExchangeValue
```

Comment:
```suggestion
	return xc != values.DefaultExchangeValue
```

Reply:
I know it probably should never be empty, but if it is, then exchange rate fetching is disabled. That's why I added the `!= ""` check.